### DISCUSSION TITLE
Is there a config file?  I can't load the web portal at my network.

### DISCUSSION THREAD

**Discussion by tazz131** [Geektoolkit/Dynaframe3#185] [Q&A] [answered]
Hello,
I'm running my Pi and DynaFrame on a school network (digital media and film teacher here) and I'm unable to SSH or get access to the DynaFrame Web Portal due to IT restrictions.

Is there a config file for DynaFrame that will allow me to make certain changes, to scaling, etc?

My current solution is to take it home whenever I need to make changes - And I just realized that it doesn't scale properly after my initial setup.

Thanks.

**Comment by qwksilver** [comment, marked as answer]
Depending on those restrictions a viable alternate might be to establish a hotspot with your phone, and then configure to access that hotspot, and with a laptop sign in to the closed network to do configuration.  

Much of the config is accessed in the web interface if that isn't locked down by the school and it's on the local network there.

so question session about restrictions in place at the school

- can you connect via hardwire to the school at all
- can you connect via wireless to the school at all
- can you have outside wireless networks(temporary) that don't access internet or school on premises.
- can you have a peer network(no wireless, no internet) on the school grounds.


as for a direct answer to the question, you can config with keyboard connected to the pi, you need to (esc) the dynaframe, and edit the appsettings.json file to change some of those options.  it's will be cumbersome.

optionally while connected to a keyboard, you can hit the windows/command key on the keyboard and launch the internet/chromium browser in the pi menu, and navigate to 127.0.0.1:80.  This will allow you to run the browser on top of dynaframe, and change settings directly in the menus that way.  I just tested this on mine, and it worked quite well.  The big secret here is that you can't actually quit the frame by hitting ESC or the web configurator won't be running as it's the same process as the frame playback.

some config options, like the playlist, are nearly impossible to config at all without a web access of some kind.
i personally use vnc, if that protocol isn't locked out, for most of my needs.

**Reply by tazz131** [reply]
Excellent. I went with opening a browser while DynaFrame was running. Connecting to the IP you provided for me into the web portal. All is well now. Thank you.
